homework_19_11/src/Triangle.java

```diff
@@ -0,0 +1,18 @@
+public class Triangle {
+    float SideOne;
+    float SideTwo;
+    float SideThree;
+
+    double Square;
+    float Perimeter;
+
+    Triangle (float SideOne, float SideTwo, float SideThree) {
+        this.SideOne = SideOne;
+        this.SideTwo = SideTwo;
+        this.SideThree = SideThree;
+    }
+    public void CalculateTriangle() {
+        Perimeter = SideOne + SideTwo + SideThree;
```

Comment:
rename it to halfPerimeter and change to halfPerimeter = (sideOne + sideTwo + sideThree)/2;